From 2504aba5d9de54aef522151a86e43baeedb77436 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Tue, 7 Jun 2022 15:13:03 +0200
Subject: [PATCH] arm64: dts: qcom: cp01-c1: enable PCIe

CP01 has a M.2 Key E slot with a single lane PCIe 3.0.
So enable the node and add the PERST and WAKE GPIO-s.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq6018-cp01-c1.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

--- a/arch/arm64/boot/dts/qcom/ipq6018-cp01-c1.dts
+++ b/arch/arm64/boot/dts/qcom/ipq6018-cp01-c1.dts
@@ -124,3 +124,14 @@
 &usb2 {
 	status = "okay";
 };
+
+&pcie_phy {
+	status = "okay";
+};
+
+&pcie0 {
+	status = "okay";
+
+	perst-gpios = <&tlmm 60 GPIO_ACTIVE_LOW>;
+	wake-gpios = <&tlmm 36 GPIO_ACTIVE_HIGH>;
+};
